features/js-self-profiling.yml

```diff
@@ -1,7 +1,7 @@
-draft_date: 2024-07-11
 name: JS Self-Profiling API
-description: TODO
+description: The JS Self-Profiling API profiles the JavaScript execution on a page and provides a trace of the execution, which you can then send to a server for later analysis.
```

Comment:
Can you name `Profiler` in the description? The constructor is the entrypoint to this API.

Reply:
Duh, of course, sorry about that.